File: a/RAM.cpp
class RAM {
	
	private:
		// Atribut
		int capacity;
		int price;

	public:
		// Konstruktor
		RAM() {
			capacity = 0;
			price = 0;
		}
		
		// Setter & Getter
		void setCapacity(int capacity) {
			this->capacity = capacity;
		}
		void setPrice(int price) {
			this->price = price;
		}

		int getCapacity() {
			return capacity;
		}
		int getPrice() {
			return price;
		}

		// Method
		void printRAM() {
			cout << "- RAM -" << endl; 
			cout << "Capacity    : " << capacity << " GB" << endl; 
		}

		// Destruktor
		~RAM(){}
};

File: b/Python/RAM.py
class RAM:
	# Konstruktor
	def __init__(self):
		self.__capacity = 0
		self.__price = 0

	# Setter & Getter
	def setCapacity(self, capacity):
		self.__capacity = capacity

	def setPrice(self, price):
		self.__price = price

	def getCapacity(self):
		return self.__capacity

	def getPrice(self):
		return self.__price

	# Method
	def printRAM(self):
		print("- RAM -")
		print("Capacity    :", self.__capacity, "GB")